content/en/docs/installing-on-linux/introduction/intro.md

```diff
@@ -81,13 +60,9 @@ KubeSphere has decoupled some core feature components since v2.1.0. These compon
 
 ## Storage Configuration Instruction
 
-The following links explain how to configure different types of persistent storage services. Please refer to [Storage Configuration Instruction](../storage-configuration) for detailed instructions regarding how to configure the storage class in KubeSphere.
+KubeSphere allows you to configure persistent storage services before installation and after installation. Meanwhile KubeSphere supports a varity of storage services including open source solutions such as Ceph, GlusterFS, etc. and commercial storage products as well. Please refer to [Persistent Storage Configurations](../storage-configuration) for detailed instructions regarding how to configure the storage class before installing KubeSphere.
```

Comment:
Note that "etc." should never be used after "include", "such as" or "for example". These terms make etc. redundant as they already imply that the writer could offer other examples. Please change this paragraph to:

KubeSphere allows you to configure persistent storage services both before installation and after installation. Meanwhile, KubeSphere supports a variety of open-source storage solutions (for example, Ceph and GlusterFS) as well as commercial storage products. Please refer to [Persistent Storage Configurations](../storage-configuration) for detailed instructions regarding how to configure the storage class before you install KubeSphere.

Reply:
done